tests/unit/CXXTests/testSidreStorage.cc

```diff
@@ -1,16 +0,0 @@
-#include "testSidreStorage.hh"
```

Comment:
I got rid of SidreDataCollection as a whole because there was nothing that was using it. I spoke to Brody and he said that we wouldn't need to communicate any data other than restarts. So I took it all out for now, can always find it on git later if we end up needing it.

Reply:
My only concern is testing Spheral data types against Sidre directly. If new types are added in the future they may slip past as the aren't used in the sidre functional restart tests.

Reply:
I'm not really understanding what you are saying. There are unit test for all of the FileIO stuff as well as functional tests that SidreFileIO uses as well. Is there something more that we should be testing for?

Reply:
Resolved offline.